website/src/data/users.js

```diff
@@ -420,6 +420,14 @@ const users = [
     source: 'https://github.com/miroiu/nodify',
     tags: ['design'],
   },
+  {
+    title: 'OCPeasy',
+    description: 'Open-source software provisioning, configuration management, and application-deployment tool enabling infrastructure as code on OpenShift.',
+    preview: require('./showcase/ocpeasy.png'),
+    website: 'https://www.ocpeasy.org',
+    source: 'https://github.com/ocpeasy/website',
+    tags: ['design'],
```

Comment:
```suggestion
    tags: [],
```

removing this as it's reserved for sites having a custom design

Reply:
I'm not able to commit to your PR so please accept my suggestion so that I can merge

Reply:
going to merge and fix this in another pr

Reply:
@slorber Just saw your message, thanks for that!